tests/test_manage_resource_path.py

```diff
@@ -43,7 +325,34 @@ def test_create_request_without_username(client):
     }
 
 
-def test_create_duplicate_request(client):
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            # resource_path
+            "username": "requestor_user",
+            "resource_path": "/my/resource",
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+        },
+        {
+            # resource_paths
+            "username": "requestor_user",
+            "resource_paths": ["/study/123456", "/study/7890", "/another_study/98765"],
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+        },
+        {
+            # resource_paths will take precedence over resource_path
+            "username": "requestor_user",
+            "resource_path": "/older_study/000111",
+            "resource_paths": ["/study/123456", "/study/7890", "/another_study/98765"],
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+        },
+    ],
+)
+def test_create_duplicate_request(client, data):
```

Comment:
you added the `data` parameter but forgot to remove `data = {...}` in the body of the test. but i'm not sure we need to add `resource_path/resource_paths` test cases to this test anyway?